docs/guidance/gcs-ha.md

```diff
@@ -0,0 +1,95 @@
+## Ray GCS HA (Experimental)
+
+> Note: This feature is still experimental, there are a few limitations and stabilization will be done in future release from both Ray and KubeRay side.
+
+### Prerequisite
+
+* Ray 2.0 is required.
+* You need to support external Redis server for Ray. (Redis HA cluster is highly recommended.)
+
+### Enable Ray GCS HA
+
+To enable Ray GCS HA in your newly KubeRay-managed Ray cluster, you need to enable it by add an annotation to the
+RayCluster YAML file.
+
+```yaml
+...
+kind: RayCluster
+metadata:
+  annotations:
+    ray.io/ha-enabled: "true" # <- add this annotation enable GCS HA
+    ray.io/external-storage-namespace: "my-raycluster-storage-namespace" # <- optional, to specify the external storage namespace
+...
+```
+An example can be found at [ray-cluster.external-redis.yaml](../../ray-operator/config/samples/ray-cluster.external-redis.yaml)
+
+When annotation `ray.io/ha-enabled` is added with a `true` value, KubeRay will enable Ray GCS HA feature. This feature
```

Comment:
Should we describe here what GCS HA does (e.g. recovery of detached actors etc.)?